File: app/src/main/java/com/einthefrog/historyquiz/MainActivity.java
package com.einthefrog.historyquiz;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.view.LayoutInflater;

import com.einthefrog.historyquiz.databinding.ActivityMainBinding;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        ActivityMainBinding binding = ActivityMainBinding.inflate(getLayoutInflater());
        setContentView(binding.getRoot());
    }
}

File: app/src/main/java/com/einthefrog/historyquiz/MainActivity.kt
package com.einthefrog.historyquiz

import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity
import com.einthefrog.historyquiz.databinding.ActivityMainBinding

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        val binding: ActivityMainBinding = ActivityMainBinding.inflate(layoutInflater)
        setContentView(binding.root)
    }
}
